### PR TITLE
doc: continue with doc organization and process improvement

### PR DESCRIPTION
**Pull request by dbkinder**
* extract current CODEOWNERS content during docbuild
* update introduction
* change :repo_file: role to :repo_blob: for GitHub name consistency

### CODE REVIEW

File: codeowner.md [hunk 1]
```diff
@@ -1,50 +1,32 @@
...
+
+<!-- codeowners.txt is generated at docbuild time from the CODEOWNERS files from each repo -->
+```{include} codeowners.txt
+```
+
+-----
 
 ## GenAIEval
 
-lvliang-intel, changwangss, lkk12014402
```
Comment (kevinintel) [inline codeowner.md:40]: please link to CODEOWNER file of each repo

Comment (dbkinder) [reply codeowner.md:40]: They will be automatically brought in during the doc build process (if the CODEOWNERS file exists).

File: examples/ChatQnA/ChatQnA_Guide.rst [hunk 2]
```diff
@@ -4,65 +4,80 @@ ChatQnA Sample Guide
...
-
-      Such as..
++--------------------------------------------------------------------------------------+
+| Single Node                                                                          |
+|                                                                                      |
++============================================+=========================================+
+| XEON Scalable Processors                   |Gaudi Servers                            |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+| NNIDIA GPUs                                | AI PC                                   |
```
Comment (mkbhanda) [inline examples/ChatQnA/ChatQnA_Guide.rst:59]: Looks like a typo NNIDIA /NVIDIA?

File: scripts/fix-github-md-refs.sh [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
 
-files=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
+# look for markdown files containing a hard github.com/opea-project/...
+# reference to a markdown file
+
+mdfiles=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
 
-# fix references to opea-project/blob/main/.../*.md to be to the repo name and
-# subsequent path to the md file
+# fix references to opea-project/tree/main/.../*.md (or blob/.../*.md to be to the repo name and
```
Comment (mkbhanda) [inline scripts/fix-github-md-refs.sh:20]: are missing a ")"

File: scripts/fix-github-md-refs.sh [hunk 4]
```diff
@@ -1,28 +1,47 @@
...
-# (as done in the makefile)
+# (as done in the makefile).  Normally is _build/rst
 
 cd ${1:-.}
 
-files=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
+# look for markdown files containing a hard github.com/opea-project/...
+# reference to a markdown file
+
+mdfiles=`grep -ril --include="*.md" 'github.com/opea-project.*\/[^\)]*\.md'`
```
Comment (mkbhanda) [inline scripts/fix-github-md-refs.sh:18]: are we missing a "(" in the regular expression?

**Comment by chickenrae** [comment]
GTG

**Review by chickenrae — APPROVED** [review]

**Review by mkbhanda — APPROVED** [review]

File: examples/ChatQnA/ChatQnA_Guide.rst (hunk 2 above)
Comment (dbkinder) [reply examples/ChatQnA/ChatQnA_Guide.rst:59]: There are lots of typos in this place holder doc that will be updated later :)

File: scripts/fix-github-md-refs.sh (hunk 4 above)
Comment (dbkinder) [reply scripts/fix-github-md-refs.sh:18]: Nope.  That pattern is looking for a string of characters that's not an open parenthesis.

File: scripts/fix-github-md-refs.sh (hunk 3 above)
Comment (dbkinder) [reply scripts/fix-github-md-refs.sh:20]: Yes, there's a missing ) in the comment...

**Pull request merged**